bitcoin/src/pow.rs

```diff
@@ -130,14 +130,14 @@ impl Sub for Work {
 /// a block. (See also [`Work`].)
 ///
 /// ref: <https://en.bitcoin.it/wiki/Target>
-#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Default)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde", serde(crate = "actual_serde"))]
 pub struct Target(U256);
 
 impl Target {
-    // When parsing nBits, Bitcoin Core converts a negative target threshold into a target of zero.
-    const ZERO: Target = Target(U256::ZERO);
+    /// When parsing nBits, Bitcoin Core converts a negative target threshold into a target of zero.
+    pub const ZERO: Target = Target(U256::ZERO);
```

Comment:
Shouldn't this be `pub(crate)` instead? Zero target seems very strange to use by consumers and we only use it in tests.

Reply:
Since it's used as a clamp in consensus code it may be useful for users who want to compare against it, or who want to test their own block-handling code in pathological cases.

It's clearly commented as being a degenerate/edge case. I think it's ok.

Reply:
Yeah users wanting to compare to zero was what made me make it public.